docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -94,24 +94,34 @@ all labeled and additional information will be included below.
 .. image:: pregnancy_decision_tree_vr.svg
 
 .. list-table:: Pregnancy Decision Tree
-  :widths: 3 15 15
+  :widths: 3 5 10 10 15
```

Comment:
Is there a way to make this table narrower? It's really wide on my screen, making the Notes column difficult to read. If the URLs are making the Source column too wide, you could maybe replace them with hyperlinked text that can be shorter and contain spaces, instead of the raw URL.

Reply:
Replaced all links with hyperlinks. Not sure how else to make it shorter...